tests/scenarios/static-app-test.ts

```diff
@@ -349,7 +349,7 @@ wideAppScenarios
               {
                 package: 'app-template',
                 appModules: {
-                  'components/fancy-box.js': {
+                  'app/components/fancy-box.js': {
```

Comment:
I'm not sure if this is the right change 🤔 one side of me thinks it should be the full path (like this change) but on the other hand it would mean that people upgrading will have to update their packageRules.

It seems like a reasonable breaking change but I could be convinced either way 🤷

Reply:
Yeah I think we should address this. The spelling of the packageRules shouldn't be changing.